src/main/java/kitchenpos/products/tobe/application/ProductService.java

```diff
@@ -0,0 +1,64 @@
+package kitchenpos.products.tobe.application;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+import kitchenpos.menus.domain.Menu;
+import kitchenpos.menus.domain.MenuProduct;
+import kitchenpos.menus.domain.MenuRepository;
+import kitchenpos.products.tobe.domain.Product;
+import kitchenpos.products.tobe.infra.ProductRepository;
+import kitchenpos.products.tobe.infra.PurgomalumClient;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class ProductService {
+    private final ProductRepository productRepository;
+    private final MenuRepository menuRepository;
+    private final PurgomalumClient purgomalumClient;
+
+    public ProductService(
+        final ProductRepository productRepository,
+        final MenuRepository menuRepository,
+        final PurgomalumClient purgomalumClient
+    ) {
+        this.productRepository = productRepository;
+        this.menuRepository = menuRepository;
+        this.purgomalumClient = purgomalumClient;
+    }
+
+    @Transactional
+    public ProductCreationResponseDto create(final String name, final BigDecimal price) {
+        final Product product = productRepository.save(Product.create(name, price, purgomalumClient));
+        return ProductCreationResponseDto.of(product);
+    }
+
+    @Transactional
+    public Product changePrice(final UUID productId, final BigDecimal price) {
```

Comment:
도메인 모델 반환 대신 DTO를 반환하게 수정하시면 좋을 것 같습니다. 😄 
```suggestion
    public ProductResponse changePrice(final ProductRequest request) {
```